frontend/src/redux/museumArtworkActions.js

```diff
@@ -0,0 +1,92 @@
+const getIIIFLevel = (artwork, displayWidth) => {
+  return {
+    url: 'https://www.artic.edu/iiif/2/' + artwork.image_id + '/full/' + displayWidth + ',/0/default.jpg',
+    width: displayWidth,
+    height: Math.floor(artwork.thumbnail.height *
+      displayWidth / artwork.thumbnail.width),
+  };
+};
+
+const convertToArtworkInfo = (artwork) => {
+  const artworkInfo = new Map();
+  artworkInfo.set('alt', artwork.thumbnail.alt_text);
+  artworkInfo.set('url', getIIIFLevel(artwork, 500).url);
+  artworkInfo.set('title', artwork.title);
+  return artworkInfo;
+};
+
+function getMuseumArtworks(path, params) {
+  const apiUrl = new URL('https://aggregator-data.artic.edu/api/v1/');
+  apiUrl.pathname += path;
+  if (params) {
+    for (const [key, value] of params) {
+      apiUrl.searchParams.append(key, value);
+    }
+  }
+  const API = apiUrl.toString();
+  return fetch(API, {
+    headers: {
+      'Content-Type': 'application/json',
+      'Accept': 'application/json',
+    },
+  })
+      .then(handleErrors)
+      .then((res) => res.json());
+}
+
+function filterArtworks(artworks) {
+  const artworksArr = [];
+  for (const artwork of artworks) {
+    if (artwork.thumbnail != null) {
+      artworksArr.push(convertToArtworkInfo(artwork));
+    }
+  }
+  return artworksArr;
+}
+
+export function fetchMuseumArtworks() {
+  return (dispatch) => {
+    dispatch(fetchMuseumArtworksBegin());
+    return getMuseumArtworks('artworks', new Map([['limit', 9]]))
```

Comment:
I believe the Map can use builder syntax to be more fluent:

```suggestion
    return getMuseumArtworks('artworks', new Map().set('limit', 9))
```